src/types/types.ts

```diff
@@ -108,6 +108,9 @@ export type HybridSearch = {
   semanticRatio?: number;
 };
 
+// @TODO: Add documentation link to available locales
```

Comment:
Link will be https://www.meilisearch.com/docs/reference/api/settings#localized-attributes